CMakeLists.txt

```diff
@@ -93,6 +113,7 @@ if(NOT SwiftFoundation_MODULE_TRIPLE)
 endif()
 
 # System dependencies
+find_package(LibRT)
```

Comment:
Copying my comment over from https://github.com/apple/swift-corelibs-foundation/pull/5056#discussion_r1706335870:

>For my context, is there a reason that's preventing us from fixing this on the dispatch side instead of applying a workaround to Foundation (and presumably any other project that imports dispatch via CMake in the toolchain)? Is there a way we can just add this to the exported config for dispatch?

Reply:
Ok thanks for the explanation, this seems fine for now then